### ISSUE TITLE
1.1.8.3 maven availability

### ISSUE THREAD

**Issue by bokken** [xerial/snappy-java#275]
@xerial, thanks for releasing 1.1.8.3. Any idea when we can expect that to be available from maven central? Did something fail during the release process? Usually the releases are available pretty quickly, but I am still not seeing it there:
https://repo1.maven.org/maven2/org/xerial/snappy/snappy-java/

**Comment by xerial** [comment]
Let me check. There might be something wrong at sonatype side

**Comment by xerial** [comment]
Sonatype OSS repository is experiencing an outage https://status.maven.org/incidents/stmcw22knx9c

I'm not sure it's related to this missing jar, but let me try uploading the jar again.

**Comment by xerial** [comment]
It looks like snappy-java 1.1.8.3 artifacts are already uploaded to Sonatype, but it's stuck at Sonatype side. We may need to wait for a while until their queue will be processed:
```
2021-01-21 11:41:17.466-0800  info [SonatypeClient] Promoting staging repository [orgxerial-1366] status:closed, profile:org.xerial(37f420f8262891) description: [sbt-sonatype] snappy-java 1.1.8.3  - (SonatypeClient.scala:218)
2021-01-21 11:41:17.706-0800  info [SonatypeClient] The promote stage is in progress. Waiting for 3.00 sec.  - (SonatypeClient.scala:172)
2021-01-21 11:41:20.807-0800  info [SonatypeClient] The promote stage is in progress. Waiting for 4.50 sec.  - (SonatypeClient.scala:172)
2021-01-21 11:41:25.406-0800  info [SonatypeClient] The promote stage is in progress. Waiting for 6.75 sec.  - (SonatypeClient.scala:172)
2021-01-21 11:41:32.420-0800  info [SonatypeClient] Activity name:release, started:2021-01-21T19:41:18.736Z  - (SonatypeClient.scala:393)
2021-01-21 11:41:32.420-0800  info [SonatypeClient] The promote stage is in progress. Waiting for 10.13 sec.  - (SonatypeClient.scala:172)
2021-01-21 11:41:43.048-0800  info [SonatypeClient]   Evaluate: id:nx-internal-ruleset, rule:RepositoryWritePolicy  - (SonatypeClient.scala:382)
2021-01-21 11:41:43.049-0800  info [SonatypeClient]   Evaluate: RepositoryWritePolicy  - (SonatypeClient.scala:382)
2021-01-21 11:41:43.051-0800  info [SonatypeClient] The promote stage is in progress. Waiting for 15.00 sec.  - (SonatypeClient.scala:172)
2021-01-21 11:41:58.322-0800  info [SonatypeClient]     Failed: RepositoryWritePolicy, failureMessage:Artifact updating: Repository ='releases:Releases' does not allow updating artifact='/org/xerial/snappy/snappy-java/1.1.8.3/snappy-java-1.1.8.3.jar'  - (SonatypeClient.scala:382)
2021-01-21 11:41:58.323-0800  info [SonatypeClient]     Failed: id:nx-internal-ruleset, failureCount:1  - (SonatypeClient.scala:382)
2021-01-21 11:41:58.323-0800  info [SonatypeClient] repositoryReleaseFailed: id:orgxerial-1366, cause:com.sonatype.nexus.staging.StagingRulesFailedException: One or more rules have failed  - (SonatypeClient.scala:382)
2021-01-21 11:41:58.323-0800 error [SonatypeClient] [promote] Failed  - (SonatypeClient.scala:184)
2021-01-21 11:41:58.323-0800 error [SonatypeClient] Activity name:release, started:2021-01-21T19:41:18.736Z  - (SonatypeClient.scala:462)
2021-01-21 11:41:58.324-0800 error [SonatypeClient]     Failed: RepositoryWritePolicy, failureMessage:Artifact updating: Repository ='releases:Releases' does not allow updating artifact='/org/xerial/snappy/snappy-java/1.1.8.3/snappy-java-1.1.8.3.jar'  - (SonatypeClient.scala:380)
2021-01-21 11:41:58.325-0800 error [Sonatype] [STAGE_FAILURE] Failed to promote the repository.  - (Sonatype.scala:406)
sbt:snappy-java>
```

**Comment by bokken** [comment]
@xerial can you try deploying again? Sonatype is reporting everything is resolved, but I still do not see 1.1.8.3 on maven central.

Thanks,
Brett

**Comment by xerial** [comment]
A problem is snappy-java 1.1.8.3 is properly deployed to their release repository, but it's not been synchronized to the Maven central repository. So I've lost a way to deploy 1.1.8.3 as Sonatype doesn't allow overwriting already uploaded release version. 

I'm going to release 1.1.8.4 as a workaround.

**Comment by xerial** [comment]
Just released 1.1.8.4 instead. And filed an issue ticket at Sonatype https://issues.sonatype.org/browse/MVNCENTRAL-6483

**Comment by bokken** [comment]
Thanks @xerial!
